analyze.go

```diff
@@ -737,6 +737,33 @@ var directives = map[string][]uint{
 	"http2_recv_timeout": {
 		ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfTake1,
 	},
+	"http3": {
```

Comment:
Could be useful to link the directives documentation somewhere nearby for historical reference?

Reply:
There is a general doc reference at the top of this list.